src/server/mirserver-internal.pc.in

```diff
@@ -0,0 +1,8 @@
+prefix=@CMAKE_INSTALL_PREFIX@
+includedir=@PKGCONFIG_INCLUDEDIR@
+
+Name: mirserver-internal
+Description: Mir server internal headers
+Version: @MIR_VERSION@
+Requires.private: mircommon
+Cflags: -I${includedir}/mirserver-internal -I${includedir}/mircommon -I${includedir}/mirplatform -I${includedir}/mircore
```

Comment:
You shouldn't add header directories from other .pcs directly - just add those packages to `Requires:`
```suggestion
Requires: mirserver mircommon mirplatform mircore
Cflags: -I${includedir}/mirserver-internal
```